simulators/utils.py

```diff
@@ -53,6 +53,11 @@ def int_to_twos(val):
 
     >>> int_to_twos(-1625)
     '11111111111111111111100110100111'
+
+    >>> int_to_twos(4294967295)
+    Traceback (most recent call last):
+        ...
+    ValueError: Out of range value, got 4294967295, expected from -2147483648 to 2147483647.
```

Comment:
What do you think if we change the message to the following?

```
raise ValueError(
             "%d out of range (%d, %d)."
             % (val, -2147483648, 2147483647)
         )
```

In this case the the traceback will be more compact:

```
"%d out of range (%d, %d)."
ValueError: 4294967295 out of range (-2147483648, 2147483647).
```

In case you agree, apply the change to utils.py directly on this pull request.